src/test/assembly/asm/hexagon-hvx64.rs

```diff
@@ -0,0 +1,80 @@
+// no-system-llvm
+// assembly-output: emit-asm
+// compile-flags: --target hexagon-unknown-linux-musl
+// compile-flags: -C target-feature=+hvx
+// compile-flags: -C target-feature=+hvx-length64b
```

Comment:
Consider merging this with the other file. See how it is done in riscv-types.rs which supports both riscv32 and riscv64.